### PR TITLE
Release 1.6.0

### PR DESCRIPTION
**Pull request by phanikumv**
Update `setup.cfg`, `docs/conf.py` and `CHANGELOG.rst` with the changes pertaining to release 1.6.0

### CODE REVIEW

**Review by bharanidharan14 — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,60 @@
...
+
+
+Enhancements
+""""""""""""
+
+- Implement OpenLineage custom extractor for BigQuery Async Operators
+  (`#429 <https://github.com/astronomer/astronomer-providers/pull/429>`_)
+- Add session specific query tag and OpenLineage Extractor for Snowflake Async operator
+  (`#437 <https://github.com/astronomer/astronomer-providers/pull/437>`_)
+
```
Comment (pankajastro) [inline CHANGELOG.rst:51]: This was also an improvement in dataproc hook. though, I'm not sure if we need to put it here or not https://github.com/astronomer/astronomer-providers/pull/453

Comment (phanikumv) [reply CHANGELOG.rst:51]: That doesnt change the way users use the dataproc submit operator, its more of a technical change. Hence in my opinion, it doesnt need to be included

**Review by pankajkoti — COMMENTED** [review]
Should we also include RedshiftDataOperatorAsync? PR: https://github.com/astronomer/astronomer-providers/pull/467

**Comment by kaxil** [comment]
> Should we also include RedshiftDataOperatorAsync? PR: #467

oh yes, we should

File: CHANGELOG.rst (hunk 1 above)
Comment (kaxil) [reply CHANGELOG.rst:51]: We can include https://github.com/astronomer/astronomer-providers/pull/448 as an improvement

**Review by pankajastro — APPROVED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (phanikumv) [reply CHANGELOG.rst:60]: included in [db210fb](https://github.com/astronomer/astronomer-providers/pull/480/commits/db210fb370f0f088e01ad7fd9c785196eed831fd)

**Comment by phanikumv** [comment]
> > Should we also include RedshiftDataOperatorAsync? PR: #467
> 
> oh yes, we should

Included in [db210fb](https://github.com/astronomer/astronomer-providers/pull/480/commits/db210fb370f0f088e01ad7fd9c785196eed831fd)

**Review by kaxil — APPROVED** [review]

**Review by bharanidharan14 — APPROVED** [review]

**Pull request merged**